### PR TITLE
Update develop-in-the-cloud.md

### PR DESCRIPTION
**Pull request by mirnawong1**
updating ide docs to include some features that affect development. see [slack thread](https://dbt-labs.slack.com/archives/C043H8Z0P7H/p1678724314676849?thread_ts=1678122006.588049&cid=C043H8Z0P7H)

added:
- individual git commit/revert files
- yaml autocomplete
- 2+models

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/get-started/develop-in-the-cloud.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ With the Cloud IDE, you can:
...
     * See Python models in DAG in dbt version 1.3 and higher
     * Currently, you can't preview python models
-- Visualize a directed acyclic graph (DAG), and more.
+- Visualize a directed acyclic graph (DAG), and [more](#cloud-ide-features).
 
-<Lightbox src src="/img/docs/dbt-cloud/cloud-ide/cloud-ide-v2.jpg" title="The dbt Cloud IDE in dark mode"/>
+<Lightbox src src="/img/docs/dbt-cloud/cloud-ide/cloud-ide-v2.jpg" width="85%" title="The dbt Cloud IDE with all its features is visible in dark mode."/>
 
 
 ## Prerequisites
```
Comment (mirnawong1) [inline website/docs/docs/get-started/develop-in-the-cloud.md]: hey @bossnunta - one of the prerequisites is dbt version 0.15.0 or higher (see below), is this now incorrect due to the <1.0 deprecation?

>Your dbt project must be compatible with dbt version 0.15.0 or higher. The dbt IDE is powered by the [dbt-rpc](https://github.com/dbt-labs/docs.getdbt.com/blob/31ce8f4e4ec01b82d7e6c5aff8b1af1befe832de/reference/commands/rpc) which was overhauled in dbt v0.15.0

Comment (bossnunta) [reply website/docs/docs/get-started/develop-in-the-cloud.md]: yeah let's just remove that line entirely! thanks for noting this mirna

File: website/docs/docs/cloud/develop-in-the-cloud.md [hunk 2]
```diff
@@ -1,51 +1,66 @@
 ---
-title: "dbt Cloud IDE"
+title: "dbt Cloud IDE""
```
Comment (mirnawong1) [inline website/docs/docs/cloud/develop-in-the-cloud.md]:
```suggestion
title: "dbt Cloud IDE"
```

Comment (mirnawong1) [inline website/docs/docs/cloud/develop-in-the-cloud.md]:
```suggestion
title: "dbt Cloud IDE"
```

File: website/docs/docs/cloud/develop-in-the-cloud.md [hunk 3]
```diff
@@ -7,45 +7,60 @@ sidebar_label: dbt Cloud IDE
...
 
 To develop in the Cloud IDE, make sure you have the following:
 
-- Your dbt project must be compatible with dbt version 0.15.0 or higher. The dbt IDE is powered by the [dbt-rpc](/reference/commands/rpc) which was overhauled in dbt v0.15.0
-- A [dbt Cloud account](https://cloud.getdbt.com/) and [Developer seat license](/docs/cloud/manage-access/seats-and-users)
-- A git repository set up and git provider must have `write` access enabled. See [Connecting your GitHub Account](/docs/cloud/git/connect-github) or [Importing a project by git URL](/docs/cloud/git/import-a-project-by-git-url) for detailed setup instructions
-- A dbt project connected to a [data platform](/docs/cloud/connect-data-platform/connect-your-database)
-- Set up a [development environment and development credentials](#access-the-cloud-ide)
-- The environment must be on dbt version 1.0 or higher
+- A [dbt Cloud account](https://cloud.getdbt.com/) and [Developer seat license](/docs/collaborate/manage-access/seats-and-users)
```
Comment (mirnawong1) [inline website/docs/docs/cloud/develop-in-the-cloud.md]:
```suggestion
- A [dbt Cloud account](https://cloud.getdbt.com/) and [Developer seat license](/docs/cloud/manage-access/seats-and-users)
```

File: website/docs/docs/cloud/develop-in-the-cloud.md [hunk 4]
```diff
@@ -7,45 +7,60 @@ sidebar_label: dbt Cloud IDE
...
+- A git repository set up and git provider must have `write` access enabled. See [Connecting your GitHub Account](/docs/collaborate/git/connect-github) or [Importing a project by git URL](/docs/collaborate/git/import-a-project-by-git-url) for detailed setup instructions
```
Comment (mirnawong1) [inline website/docs/docs/cloud/develop-in-the-cloud.md]:
```suggestion
- A git repository set up and git provider must have `write` access enabled. See [Connecting your GitHub Account](/docs/cloud/git/connect-github) or [Importing a project by git URL](/docs/cloud/git/import-a-project-by-git-url) for detailed setup instructions
```

File: website/docs/docs/cloud/develop-in-the-cloud.md [hunk 5]
```diff
@@ -7,45 +7,60 @@ sidebar_label: dbt Cloud IDE
...
+- A dbt project connected to a [data platform](/docs/get-started/connect-your-database)
```
Comment (mirnawong1) [inline website/docs/docs/cloud/develop-in-the-cloud.md]:
```suggestion
- A dbt project connected to a [data platform](/docs/cloud/connect-data-platform/connect-your-database)
```

File: website/docs/docs/cloud/develop-in-the-cloud.md [hunk 6]
```diff
@@ -7,45 +7,60 @@ sidebar_label: dbt Cloud IDE
...
+- Set up a [development environment and development credentials](#access-the-cloud-ide) on dbt version 1.0 or higher
```
Comment (mirnawong1) [inline website/docs/docs/cloud/develop-in-the-cloud.md]:
```suggestion
- Set up a [development environment and development credentials](#access-the-cloud-ide)
- The environment must be on dbt version 1.0 or higher
```

File: website/docs/docs/cloud/develop-in-the-cloud.md [hunk 7]
```diff
@@ -20,32 +36,32 @@ To develop in the Cloud IDE, make sure you have the following:
 
 ## Cloud IDE features
 
-The dbt Cloud IDE comes with [tips](/docs/cloud/dbt-cloud-tips) and features that make it easier for you to develop, build, compile, run, and test data models. 
+The dbt Cloud IDE comes with [tips](/docs/get-started/dbt-cloud-tips) and features that make it easier for you to develop, build, compile, run, and test data models.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/develop-in-the-cloud.md]:
```suggestion
The dbt Cloud IDE comes with [tips](/docs/cloud/dbt-cloud-tips) and features that make it easier for you to develop, build, compile, run, and test data models. 
```

File: website/docs/docs/cloud/develop-in-the-cloud.md [hunk 8]
```diff
@@ -26,26 +42,26 @@ Read more about IDE updates in the [dbt Cloud release notes](/docs/dbt-versions/
 
 | Feature  |  Info |
 |---|---|
-| **Keyboard shortcuts** | You can access a variety of [commands and actions](/docs/cloud/dbt-cloud-tips#cloud-ide-keyboard-shortcuts) in the IDE by choosing the appropriate keyboard shortcut. This includes the command palette (Command-P or Control-P) which has common shortcuts and build commands to optimize your developer workflow, like building modified models or continuing the build from the last failure. |
+| **Keyboard shortcuts** | You can access a variety of [commands and actions](/docs/get-started/dbt-cloud-tips#cloud-ide-keyboard-shortcuts) in the IDE by choosing the appropriate keyboard shortcut. This includes the command palette (Command-P or Control-P), which has common shortcuts and build commands to optimize your developer workflow. For example, you can build modified models or continue the build from the last failure point. |
```
Comment (mirnawong1) [inline website/docs/docs/cloud/develop-in-the-cloud.md]:
```suggestion
| **Keyboard shortcuts** | You can access a variety of [commands and actions](/docs/cloud/dbt-cloud-tips#cloud-ide-keyboard-shortcuts) in the IDE by choosing the appropriate keyboard shortcut. This includes the command palette (Command-P or Control-P), which has common shortcuts and build commands to optimize your developer workflow. For example, you can build modified models or continue the build from the last failure point. |
```

File: website/docs/docs/cloud/develop-in-the-cloud.md [hunk 9]
```diff
@@ -75,13 +91,13 @@ The IDE uses developer credentials to connect to your data platform. These devel
 
 Set up your developer credentials:
 
-1. Navigate to your **Credentials** under **Your Profile** settings, which you can access at `https://YOUR_ACCESS_URL/settings/profile#credentials`, replacing `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan.
+1. Navigate to your **Credentials** under **Your Profile** settings, which you can access at `https://YOUR_ACCESS_URL/settings/profile#credentials`, replacing `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/deploy/regions-ip-addresses) for your region and plan.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/develop-in-the-cloud.md]:
```suggestion
1. Navigate to your **Credentials** under **Your Profile** settings, which you can access at `https://YOUR_ACCESS_URL/settings/profile#credentials`, replacing `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan.
```

File: website/docs/docs/cloud/develop-in-the-cloud.md [hunk 10]
```diff
@@ -106,7 +122,7 @@ The dbt Cloud IDE makes it possible to [build and view](/docs/collaborate/build-
 
 ## Related docs
 
-- [dbt Cloud features](/docs/cloud/about-cloud/dbt-cloud-features)
+- [dbt Cloud features](/docs/get-started/dbt-cloud-features)
```
Comment (mirnawong1) [inline website/docs/docs/cloud/develop-in-the-cloud.md]:
```suggestion
- [dbt Cloud features](/docs/cloud/about-cloud/dbt-cloud-features)
```

File: website/docs/faqs/API/rotate-token.md [hunk 11]
```diff
@@ -17,8 +15,8 @@ curl --location --request POST 'https://YOUR_ACCESS_URL/api/v2/users/YOUR_USER_I
 ```
 
 * Find your `YOUR_USER_ID` by reading [How to find your user ID](/faqs/Accounts/find-user-id).
-* Find your `YOUR_CURRENT_TOKEN` by going to **Account Settings** -> **API Access** and copying the API key.
-* Find [`YOUR_ACCESS_URL`](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan.
+* Find your `YOUR_CURRENT_TOKEN` by going to **Profile Settings** -> **API Access** and copying the API key.
+* Find [`YOUR_ACCESS_URL`](/docs/deploy/regions-ip-addresses) for your region and plan.
```
Comment (mirnawong1) [inline website/docs/faqs/API/rotate-token.md]:
```suggestion
* Find [`YOUR_ACCESS_URL`](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan.
```

File: website/docs/faqs/API/rotate-token.md [hunk 12]
```diff
@@ -15,7 +15,7 @@ curl --location --request POST 'https://YOUR_ACCESS_URL/api/v2/users/YOUR_USER_I
 ```
 
 * Find your `YOUR_USER_ID` by reading [How to find your user ID](/faqs/Accounts/find-user-id).
-* Find your `YOUR_CURRENT_TOKEN` by going to **Account Settings** -> **API Access** and copying the API key.
+* Find your `YOUR_CURRENT_TOKEN` by going to **Profile Settings** -> **API Access** and copying the API key.
```
Comment (runleonarun) [inline website/docs/faqs/API/rotate-token.md:18]: Nice catch!

File: website/docs/docs/cloud/develop-in-the-cloud.md [hunk 13]
```diff
@@ -26,26 +26,26 @@ Read more about IDE updates in the [dbt Cloud release notes](/docs/dbt-versions/
...
-| **IDE git button** | The git button in the IDE allows you to apply the concept of [version control](/docs/collaborate/git/version-control-basics) to your project. You can create or change branches, [resolve merge conflicts](/docs/collaborate/git/merge-conflicts), and execute git commands directly in the IDE. |
-| **Documentation** | You can generate and view your [project documentation](/docs/collaborate/build-and-view-your-docs) for your dbt project in real-time. You can inspect and verify what your project's documentation will look like before you deploy your changes to production. |
+| **IDE version control** | The IDE version control section and git button allows you to apply the concept of [version control](/docs/collaborate/git/version-control-basics) to your project directly into the IDE. <br /><br /> - Create or change branches<br /> - Commit or revert individual files by right-clicking the edited file<br /> - [Resolve merge conflicts](/docs/collaborate/git/merge-conflicts)<br /> - Execute git commands using the git button <br /> - Link to the repo directly by clicking the branch name |
```
Comment (runleonarun) [inline website/docs/docs/cloud/develop-in-the-cloud.md]:
```suggestion
| **IDE version control** | The IDE version control section and git button allow you to apply the concept of [version control](/docs/collaborate/git/version-control-basics) to your project directly into the IDE. <br /><br /> - Create or change branches<br /> - Commit or revert individual files by right-clicking the edited file<br /> - [Resolve merge conflicts](/docs/collaborate/git/merge-conflicts)<br /> - Execute git commands using the git button <br /> - Link to the repo directly by clicking the branch name |
```

**Review by runleonarun — DISMISSED** [review]
Just a tiny suggestion then it looks good to ship! 🚢

**Comment by mirnawong1** [comment]
thanks so much @runleonarun - this should be good to go now. can you please approve?

**Review by runleonarun — APPROVED** [review]
Looks great!

**Comment by mirnawong1** [comment]
woohoo! thanks @runleonarun ! cc @matthewshaver it's now in next

**Pull request merged**